src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--sun calendar__month-length--29">
```

Comment:
BEM naming convention is not properly followed. Modifiers should not be nested. It should be 'calendar--start-day--sun' and 'calendar--month-length--29'
```suggestion
    <div class="calendar--start-day--sun calendar--month-length--29">
```

Reply:
resolved